### PR TITLE
Update testing and CI

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
# What does this PR do?

Updating testing and CI

# Is there a relevant Issue open for this?
#67

### CODE REVIEW

**Comment by sean-m-sullivan** [comment]
I am not sure why the test is failing, its done that way on like 3 other repositories, I think its just not injecting the password for a silly reason.

File: meta/runtime.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
-requires_ansible: '>=2.9.6'
+requires_ansible: '>=2.11'
```
Comment (Tompage1994) [inline meta/runtime.yml]: Is this really the right thing to do? I don't think we really do require that later version of Ansible.

In another collection I've just turned off the `meta-unsupported-ansible` rule as it doesn't really make sense here.

See https://github.com/redhat-cop/ah_configuration/blob/725b92b93d275a5bf1d2de5e1d6d1993bcc596e1/.ansible-lint#L17

**Review by Tompage1994 — CHANGES_REQUESTED** [review]

File: meta/runtime.yml (hunk 1 above)
Comment (sean-m-sullivan) [reply meta/runtime.yml]: Good catch, fixed.

**Review by Tompage1994 — APPROVED** [review]

File: .github/workflows/ci_testing.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ jobs:
         run: ansible-galaxy collection install containers.podman -f
 
       - name: Docker login
-        run: docker login -u="redhat_cop+github_action" -p="${{ secrets.quay_token }}" quay.io
+        run: docker login -u="redhat_cop+gith_action" -p=${{ secrets.QUAY_TOKEN }} quay.io
```
Comment (Tompage1994) [inline .github/workflows/ci_testing.yaml]: Is this correct? the CI is failing

**Review by Tompage1994 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_testing.yaml (hunk 2 above)
Comment (sean-m-sullivan) [reply .github/workflows/ci_testing.yaml]: Its not, I was trying to fix it as was failing before, I've fixed the user name, but expect it to still fail.

**Comment by Tompage1994** [comment]
Merging despite failing CI to see if it works 🤞

**Pull request merged**